Release v5.2.0 (#312)
embrace-io/embrace-react-native-sdk: packages/react-native-otlp/README.md

```

## Initializing in the Native Layer

> If you already have the Embrace React Native SDK initialized in the Native Side or if you are planning tu run the install scripts mentioned in our docs section you could still get benefit of the OTLP custom export feature. Remember that the the install scripts are adding the minimum code needed for initializing Embrace in the Native side but it's not integrating the configuration for exporting the telemetry data into your backend of your choice. For this you would need to tweak manually both Android/iOS sides.

Reviewer: ```suggestion
If you already have the Embrace React Native SDK initialized in the Native Side or if you are planning to run the install scripts mentioned in our docs section you could still get benefit of the OTLP custom export feature. Remember that the install scripts are adding the minimum code needed for initializing Embrace in the Native side but are not integrating the configuration for exporting the telemetry data into your backend of your choice. For this you would need to tweak manually both the Android/iOS sides.
```

### iOS

> If you already run the install script mentioned above you would be able to find the `EmbraceInitializer.swift` file with some code.

Reviewer: ```suggestion
If you already ran the install script mentioned above you would be able to find the `EmbraceInitializer.swift` file with some initial code that you can update:
```

    static func start() -> Void {
        do {
          // Preparing Span Exporter config with the minimum required
          let traceExporter = OtlpHttpTraceExporter(endpoint: URL(string: GRAFANA_TRACES_ENDPOINT)!,
            config: OtlpConfiguration(
                headers: [("Authorization", GRAFANA_AUTH_TOKEN)]
            )
          )
          
          // Preparing Log Exporter config with the minimum required
          let logExporter = OtlpHttpLogExporter(endpoint: URL(string: GRAFANA_LOGS_ENDPOINT)!,
             config: OtlpConfiguration(
                headers: [("Authorization", GRAFANA_AUTH_TOKEN)]
             )
          )
          
          try Embrace
              .setup(
                  options: Embrace.Options(
                      appId: "__YOUR APP ID__",
                      platform: .reactNative,
                      export: OpenTelemetryExport(spanExporter: traceExporter, logExporter: logExporter) // passing the configuration into `export`
                  )
              )
              .start()
        } catch let e {
            print("Error starting Embrace \(e.localizedDescription)")
        }
    }
}
```

## Android

> Similar than iOS, if you already run the install script you will see the following line already in place in your `MainApplication` file.

Reviewer: ```suggestion
Similar to iOS, if you already ran the install script you will see the following line already in place in your `MainApplication` file:
```

```
Embrace.getInstance().start(this, false, Embrace.AppFramework.REACT_NATIVE)
```

> Tweak the `onCreate` method following this snippet to initialize the exporters with the minimum configuration needed. Notice that you already have all of what you need, so no extra imports are required into this file.

Reviewer: ```suggestion
Tweak the `onCreate` method using this following snippet to initialize the exporters with the minimum configuration needed. Notice that you already have all of what you need, so no extra imports are required into this file.
```
